src/test/java/org/commcare/formplayer/junit/MockTimezoneProvider.java

```diff
@@ -5,6 +5,7 @@
 /**
  * Mock timezone provider
  */
+
```

Comment:
Not for future reference, making unnecessary changes in the PR makes it harder to review.